### PR TITLE
Add string-03

### PR DESCRIPTION
**Pull request by luuhieu8720**

### CODE REVIEW

File: DotnetIntern2021/Strings/Exercise3.cs [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+using System.Text;
+using System.Threading.Tasks;
+
+namespace DotnetIntern2021.Strings
+{
+    public class Exercise3
+    {
+        public  string Reverse (string s)
+        {
+            string ans = "";
```
Comment (voquanghoa) [inline DotnetIntern2021/Strings/Exercise3.cs]: Dùng StringBuilder

Comment (luuhieu8720) [reply DotnetIntern2021/Strings/Exercise3.cs]: done

File: DotnetIntern2021Test/Strings/TestExercise3.cs [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+using NUnit.Framework;
+namespace DotnetIntern2021Test.Strings
+{
+    class TestExercise3
+    {
+        private readonly Exercise3 exercise3 = new();
+        [TestCase("abcdef","fedcba")]
+        [TestCase("","")]
+        [TestCase("123xyz","zyx321")]
+        public void Test(string s, string ans)
```
Comment (voquanghoa) [inline DotnetIntern2021Test/Strings/TestExercise3.cs]:
```suggestion
        public void Test_Reverse(string s, string ans)
```

Comment (luuhieu8720) [reply DotnetIntern2021Test/Strings/TestExercise3.cs]: done

**Pull request merged**